src/vl53l1_platform_user_config.h

```diff
@@ -30,82 +30,142 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  * @file  vl53l1_platform_user_config.h
  *
  * @brief EwokPlus compile time user modifiable configuration
+ *
+ * The default configuration values may be customized without modifying the library code by either:
+ *
+ * - Predefining the macro using a preprocessor option.
+ *   Using Arduino IDE this can be achieved with help of the
+ *   [`platform.local.txt`](https://arduino.github.io/arduino-cli/0.20/platform-specification/#platformlocaltxt)
+ *   configuration file.
+ *   For example to change the value of `VL53L1_MAX_RANGE_RESULTS` to `1` add this line to
+ *   `platform.local.txt`:
+ *   ```
+ *   compiler.cpp.extra_flags=-DVL53L1_MAX_RANGE_RESULTS=1
+ *   ```
+ * - Providing alternative macro definitions within a file named `platform_config_custom.h` which
+ *   must be locatable by the preprocessor.
+ *   For this to work the compiler must support C++17 or newer.
  */
 
 
 #ifndef _VL53L1_PLATFORM_USER_CONFIG_H_
 #define _VL53L1_PLATFORM_USER_CONFIG_H_
 
-#define    VL53L1_BYTES_PER_WORD              2
-#define    VL53L1_BYTES_PER_DWORD             4
+#if __cplusplus >= 201703L /* for C++17 or newer */
```

Comment:
I don't like this check because for example STM32duino Core that uses C++14 supports __has_include directive. It is better to replace this line with:

#if defined __has_include

Reply:
Thank you for the tip! This is much better. I had not read that part of the "fine print" of the standard (in C++17 §19.1 ¶5).

I applied the change.